api.js

```diff
@@ -41,11 +41,25 @@ module.exports = function (path, remoteApi, _remoteCall) {
           }
     })(name, api[name])
     return obj
-  })(emitter, remoteApi, path)
+  }
+
+  if (bootstrap) {
+    remoteApi['manifest'] = function () {
+      return remoteCall('async', 'manifest', [].slice.call(arguments))
+    }
+
```

Comment:
instead of adding the method then calling it, just call `remoteCall('async', 'manifest', [cb])` in the bootstrap function

Reply:
I don't actually use this, I don't think it's necessary, I think it was just to have the API there for if someone outside calls it.